.github/workflows/prod-build.yml

```diff
@@ -42,6 +42,12 @@ on:
         required: false
         default: ${DEFAULT_LOG_EACH_SUCCESSFUL_UPLOAD}
 
+      invalidate:
+        description: "Invalidate CDN (use only in exceptional circumstances)"
+        type: boolean
```

Comment:
Oh, wow, so it's actually possible to have boolean inputs. I must have stumbled upon an outdated blog article or so when I looked it up and came to the conclusion it isn't possible...